apps/etterlatte-saksbehandling-ui/client/src/components/behandling/virkningstidspunkt/utils.ts

```diff
@@ -111,6 +111,11 @@ export const OMS_OPPHOER_HJEMLER: Array<Hjemmel> = [
   },
 ]
 
+export const OMS_ALDERSOVERGANG_HJEMLER: Array<Hjemmel> = [
+  { lenke: 'https://lovdata.no/lov/1997-02-28-19/§17-11', tittel: 'Folketrygdloven § 17-11' },
+  { lenke: 'https://lovdata.no/lov/1997-02-28-19/§22-12', tittel: 'Folketrygdloven § 22-12 sjette ledd' },
```

Comment:
Vurderte å gjenbruke konstanten over for 22-12, men synes det reduserer lesbarheten betraktelig.